docusaurus/docs-internals/editor-controls/multivalue-controls/gallery.md

```diff
@@ -0,0 +1,190 @@
+| `devices?`              | `"all"` \|   `"desktop"`  \| `"responsive"`                                                                                                                                   | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`             | `boolean`                                                                                                                                                                     | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
+| `helper?.content`       | `string`                                                                                                                                                                      |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                 |
+| `helper?.position`      | `"top-start" \| "top" \| "top-end" \| "right-start" \| "right" \| "right-end" \| "bottom-end" \| "bottom" \| "bottom-start" \| "left-end" \| "left" \| "left-start" \| "top"` | `"top"` | Specifies the position of the tooltip relative to the helper icon                                                                                                                                                                 |
+| `config?.canDeleteLast` | `boolean`                                                                                                                                                                     |  `true`  | If provided and set to `false`, the user cannot delete the last item in the gallery.                                                                                                                                              |
+
+### Return value
+
+Returns a stringified array of objects with the following values:
+
+```js
+{
+  id: number;
+  uid: string;
+  fileName: string;
+  width: number;
+  height: number;
+}
+```
+Example: 
+```js
+{
+  value: [{"id":1,"uid":"dbe3a79df22c4ef2ac6553aa241d2c5c.jpg","fileName":"image1.jpg","width":1600,"height":1042},{"id":2,"uid":"a7c45ea83554c81024b84d724fd8286d.jpg","fileName":"image2.jpg","width":640,"height":359}]
+}
+
+```
+
+`id` - The ID of image on gallery (index); <br/>
+`uid` - The UID of image; <br/>
+`fileName` - The `fileName` of image; <br/>
+`width` - Width of the image; <br/>
+`height` - Height of the image; <br/>
```

Comment:
`id` - the index of the image in gallery
`uid` - the image unique identifier
`fileName` - file name of the image
`width` - image width
`height` - image height


P.S. check if was writed grammatically correct